libs/documentation/src/lib/components/tooltip-popover/demos/basic/popup-basic.component.html

```diff
@@ -5,46 +5,53 @@
     The content of this popover accepts either a string or element reference
     passed to the directive.</p>
 
-    <div style="display: flex;">
-      <div style="padding-left: 3rem;">
-        <usa-icon [icon]="'caret-down-square'" [size]="'2x'"[sdsPopover]="'Content'" [sdsPopoverTitle]="'Title'" [position]="'top'" tabindex="0" aria-label="Down carat"></usa-icon>
-      </div>
-      <div style="padding-left: 3rem;">
-        <p #title style="margin: 0em 1em;">Have a suggestion?</p>
-        <p #content style="margin: 0em 1em;">Please let us know what we could do better.</p>
-        <div class="sds-stack sds-1x" [sdsPopover]="content" [sdsPopoverTitle]="title" [position]="'right'" tabindex="0" aria-label="Thumbs down">
-          <usa-icon class="sds-stack-icon" [size]="'2x'" [icon]="'square-fill'"></usa-icon>
-          <usa-icon class="sds-stack-icon text-base-lightest" [size]="'lg'" [icon]="'hand-thumbs-down'"></usa-icon>
-        </div>
+  <div style="display: flex;">
+    <div style="padding-left: 3rem;">
+      <usa-icon [icon]="'caret-down-square'" [size]="'2x'" [sdsPopover]="'Content'" [sdsPopoverTitle]="'Title'"
+        [position]="'top'" tabindex="0" aria-label="Down carat"></usa-icon>
+    </div>
+    <div style="padding-left: 3rem;">
+      <p #title style="margin: 0em 1em;">Have a suggestion?</p>
+      <p #content style="margin: 0em 1em;">Please let us know what we could do better.</p>
+      <div class="usa-stack sds-1x" [sdsPopover]="content" [sdsPopoverTitle]="title" [position]="'right'" tabindex="0"
```

Comment:
Can you update this to use usa-stacked-icon instead? That component is meant to actually wrap usa-icon and under the hood will apply this style and any others needed.

So should be something like `<usa-stacked-icon [size]="'2x'" [sdsPopover]="content" [sdsPopoverTitle]="title" [position]="'right'" tabindex="0" aria-label="Thumbs down">`

Also, just a note, 1x is not a valid size, 'lg' or '2x' should be about the right size.

Reply:
Updated to usa-stacked-icon.